scripts/port-check/port.sh

```
@@ -0,0 +1,106 @@
+echo "Checking port usage..."
+
+
+# Loop through each port to check if it is in use
+for port in "${ports[@]}"; do
+  echo -e "\n==============================="
+  echo "  Checking port $port"
+
+ # Check for Linux systems
+  if [[ "$OS" == "Linux" ]]; then
+   # Using lsof to check port usage
+    usage=$(sudo lsof -i :$port)
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)